docs/resources/network_peering.md

```diff
@@ -397,6 +397,12 @@ Network Peering Connections can be imported using project ID and network peering
 $ terraform import mongodbatlas_network_peering.my_peering 1112222b3bf99403840e8934-5cbf563d87d9d67253be590a-AWS
 ```
 
+To obtain `project_id` and `peering_id` can be obtained using atlas cli. Attention: `atlas networking peering list` needs the `--provider` parameter for AZURE and GCP as it returns only `AWS` peerings by default. 
```

Comment:
```suggestion
To obtain your `project_id` and `peering_id` with the atlas cli. Attention: The `atlas networking peering list` command returns `AWS` peerings by default. You must include the the `--provider` parameter for AZURE and GCP peerings. 
```

Reply:
Applied the suggestion here https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2488